client/src/components/common/notifications/common.js

```diff
@@ -0,0 +1,12 @@
+const NotificationMessage = {
+  LIKED_POST: 'Your post was liked!'
+};
+
+const SocketEvent = {
```

Comment:
hmmm
we have already spilt it up

Reply:
My bad, yesterday's PR was cherry-picked from this branch's commit, so this file just wasn't deleted :c